src/main/java/com/upversionlab/controller/IngredientController.java

```diff
@@ -11,17 +12,19 @@
  * Created by rborcat on 1/3/2017.
  */
 @RestController
+@RequestMapping("/ingredients")
 public class IngredientController {
     private Map<Integer, Ingredient> ingredients = new HashMap<>();
+    private static final String INGREDIENT_RESOURCE_NAME = Ingredient.class.getSimpleName();
```

Comment:
Constants should come first (and be probably be separated by an empty line from other attributes).